tests/upgrade_from/test_upgrade.sh

```diff
@@ -117,11 +117,11 @@ install_mariadb_from_archive() {
     # The missing libraries depned on the latest distribution version, so that is found by scanning the MariaDB archive directory.
 
     log Finding MariaDB RPM repository for version $version
-    if ! $(wget --spider -r --no-parent --level 1 --quiet https://archive.mariadb.org/mariadb-$version/yum/fedora/ -P /tmp/); then
+    if ! $(wget -r --no-parent --level 1 --quiet https://archive.mariadb.org/mariadb-$version/yum/fedora/ -P /tmp/); then
         log Could not find RPMs
         exit 1
     fi
-    latest_distro=$(ls -1v /tmp/archive.mariadb.org/mariadb-$version/yum/fedora/ | tail -n 1)
+    latest_distro=$(ls -1v /tmp/archive.mariadb.org/mariadb-$version/yum/fedora/ | grep -v 'index.html' | tail -n 1)
```

Comment:
I feel like this whole section would be better as a single curl | grep | sort | tail or something similar. But this probably works too.